messaging/kafka_topics.py

```diff
@@ -0,0 +1,265 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# (c) 2015, James Bowes <jbowes@repl.ca>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = """
+---
+module: kafka_topics
+short_description: Create and modify Kafka topics.
+description:
+  - Create and modify I(Kafka) topics.
+notes:
+  - If you have multiple topics to modify, using the I(topics) option is much
+    faster, as it can execute C(kafka-topics.sh) fewer times.
+requirements:
+  - C(kafka-topics.sh) must be installed on the host.
+author: James Bowes
+options:
+  name:
+    description:
+      - The topic name to create or update
+    aliases: [topic]
+  partitions:
+    description:
+      - The number of partitions for I(name), or the default partitions for
+        I(topics) if nothing is specified.
+  replicas:
+    description:
+      - The replication factor for I(name), or the default replication factor
+        for I(topics) if nothing is specified.
+  topics:
```

Comment:
If I'm reading everything correctly, there's a special case just for package managers to do this?
- [`runner/__init__.py#L751`](https://github.com/ansible/ansible/blob/d706fd1eda7aa2b525d77edd5e6785e9708833e2/lib/ansible/runner/__init__.py#L751)
- https://github.com/ansible/ansible/issues/279

For this module, would making `name`, `topic`, and `topics` all be aliases that could either be a single item or a list? It's quite likely the created topics will have all have different values for `partition` and `replicas`, and there's not native syntax for specifying all 3 required values in a single string like there is for `apt`.